versions/3.0.md

```diff
@@ -1848,6 +1920,84 @@ breed: Mixed
 
 ```
 
+#### <a name="describedExampleObject"></a>Described Example Object
+
+##### Fixed Fields
+Field Name | Type | Description
+---|:---:|---
+<a name="describedExampleSummary"></a>summary | `string` | Short description for the example.
+<a name="describedExampleDescription"></a>description | `string` | Long description for the example.
+<a name="describedExampleValue"></a>value | [Example Object](#exampleObject) | Embedded literal example. The `value` field and `valueUrl` field are mutually exclusive.
+<a name="describedExampleValueUrl"></a>valueUrl | `string` | A URL that points to the literal example. This provides the ability to reference examples that cannot easily be included in JSON or YAML documents.  The `value` field and `valueUrl` field are mutually exclusive. 
```

Comment:
It can't be done like that as then we would lose the `summary` and `description` properties.  The alternative would be this:
```yaml
examples:
  foo: 
    summary: this is a referenced example value
    value:
      $ref: http://example.com/reuse.json#/example/one
```
The downside to this, is that you can no longer reference example values that are not JSON or YAML.  Based on the conversation with @fehguy the other day, I am recommending that we add wording to the reference object definition that $ref can only point to JSON or YAML content.
By using `valueUrl` we can have something special for examples that may be other formats.  (and later schemas, but shhhhhh, don't say that out loud).